### DISCUSSION TITLE
Setting the output Game/Window Title

### DISCUSSION THREAD

**Discussion by KingIAmStreaming** [stride3d/stride#2440] [Newcomers] [answered]
I'm learning so much so fast here and my mind is blown by how well this Game/Engine/Animation/Studio (and so much more) works, It's so responsive but.....

I looked everywhere and couldn't figure out how to set the Title of the built application if anyone could help me out while I hang my head in shame, thanks

Is this next issue a bug and should I report it? I couldn't rename anything in the solution explorer despite the option in the context menu being enabled for me to do so.

**Comment by VaclavElias** [comment, marked as answer]
This is a very nice feedback! Do catch up on our Discord server 🙂.

I do it like so in code-only. Most likely you can do the same.

```csharp
using Stride.CommunityToolkit.Engine;
using Stride.CommunityToolkit.Rendering.ProceduralModels;
using Stride.Core.Mathematics;
using Stride.Engine;

using var game = new Game();

game.Run(start: Start);

void Start(Scene rootScene)
{
    game.Window.Title = "Hey Bro!";

    game.SetupBase3DScene();

    var entity = game.Create3DPrimitive(PrimitiveModelType.Capsule);

    entity.Transform.Position = new Vector3(0, 8, 0);

    entity.Scene = rootScene;
}
```

<img width="961" alt="image" src="https://github.com/user-attachments/assets/530fa314-42f7-4796-bede-9a7c2cdf21a2">

**Comment by KingIAmStreaming** [comment]
Start.... Oh, it makes so much sense now. 
Much appreciated, Yes I am on Discord but haven't had much time to figure out what's going on there. So much for me to learn lol I can see why it's hard for people to get used to this but at the same time, for this .Net developer, this is home now :)

Note: I had been trying to set game.window.title but during game construction before calling Run I very quickly realised my mistake as Window was still null. Hunted the docs for ages but must have missed it. I had hoped to find this in the basic tutorials and get started so maybe a note for someone to do? Assuming I looked hard enough.

**Reply by VaclavElias** [reply]
No problem. Also, I am not sure if the Game Settings also have this option somewhere 

https://doc.stride3d.net/latest/en/manual/game-studio/game-settings.html

**Reply by Doprez** [reply]
It may be nice to have general windows settings available through the game settings as I dont think there is anything apart from the Render defaults. I can take a look at this when I get back to my PR for the windowing changes https://github.com/stride3d/stride/pull/2404 (Maybe this could be a bonus feature addon)

**Comment by KingIAmStreaming** [comment]
I had no luck with the game settings however I can confirm this is the way I will implement it, thanks again. I would have been hitting my head on the wall for days with out it.
 
```
using Stride.Engine;

namespace AnimationStudio.ControlCenter
{
    internal sealed class ControlCenter3DEngine : Game
    {
        internal ControlCenter3DEngine() { }

        protected override void Initialize()
        {
            base.Initialize();
            Window.Title = "Animation Studio - Control Center";
        }
    }
}
```

and updated my entry point..

```
using AnimationStudio.ControlCenter;

using var game = new ControlCenter3DEngine();
game.Run();
```
